### ISSUE TITLE
No isBuffer export in 'buffer' npm package

### ISSUE THREAD

**Issue by BensThoughts** [BensThoughts/solana-web3-angular-error#1]
Attempting to use any part of the @solana/web3.js package in an Angular 12.1.1 application leads to the error below when trying to run "ng build" or "ng serve --open".  Project generally cannot build.  This repo is a fresh "ng new" project and then a simple attempt to use @solana/web3.js to create a Keypair and log it to the console in /src/app/app.component.ts as displayed in this repo. @jstarry can you help?

```
./node_modules/@solana/web3.js/lib/index.browser.esm.js:831:6-14 - Error: export 'isBuffer' (imported as 'isBuffer') was not found in 'buffer' (possible exports: Buffer, INSPECT_MAX_BYTES, SlowBuffer, kMaxLength)

./node_modules/@solana/web3.js/lib/index.browser.esm.js:1025:13-21 - Error: export 'isBuffer' (imported as 'isBuffer') was not found in 'buffer' (possible exports: Buffer, INSPECT_MAX_BYTES, SlowBuffer, kMaxLength)

./node_modules/@solana/web3.js/lib/index.browser.esm.js:1025:33-41 - Error: export 'isBuffer' (imported as 'isBuffer') was not found in 'buffer' (possible exports: Buffer, INSPECT_MAX_BYTES, SlowBuffer, kMaxLength)

./node_modules/@solana/web3.js/lib/index.browser.esm.js:1049:13-21 - Error: export 'isBuffer' (imported as 'isBuffer') was not found in 'buffer' (possible exports: Buffer, INSPECT_MAX_BYTES, SlowBuffer, kMaxLength)

./node_modules/@solana/web3.js/lib/index.browser.esm.js:1049:34-42 - Error: export 'isBuffer' (imported as 'isBuffer') was not found in 'buffer' (possible exports: Buffer, INSPECT_MAX_BYTES, SlowBuffer, kMaxLength)
```

**Comment by jstarry** [comment]
This is getting injected by the `rollup-plugin-node-polyfills` plugin, working on removing that from web3.js, thanks for the report! I will post back here when the issue is resolved, thanks for your patience!

**Comment by jstarry** [comment]
Feel free to apply this patch to get your app working until we have new releases out:
```diff
diff --git a/package.json b/package.json
index 9e09632..901e5c3 100644
--- a/package.json
+++ b/package.json
@@ -18,8 +18,7 @@
     "@angular/platform-browser": "~12.1.0",
     "@angular/platform-browser-dynamic": "~12.1.0",
     "@angular/router": "~12.1.0",
-    "@solana/web3.js": "^1.20.0",
-    "assert": "^2.0.0",
+    "@solana/web3.js": "github:jstarry/solana-web3.js#buffer-layout",
     "rxjs": "~6.6.0",
     "tslib": "^2.2.0",
     "zone.js": "~0.11.4"

```

**Comment by BensThoughts** [comment]
Thank you @jstarry I appreciate the pre-patch solution very much.

**Comment by jstarry** [comment]
The fix is released on v1.20.2 🎉
